StereoKit/Native/NativeAPI.cs

```diff
@@ -105,8 +105,9 @@ internal static class NativeAPI
 		[DllImport(dll, CharSet = cSet, CallingConvention = call)] public static extern void   mesh_set_draw_inds(IntPtr mesh, int index_count);
 		[DllImport(dll, CharSet = cSet, CallingConvention = call)] public static extern void   mesh_set_bounds   (IntPtr mesh, in Bounds bounds);
 		[DllImport(dll, CharSet = cSet, CallingConvention = call)] public static extern Bounds mesh_get_bounds   (IntPtr mesh);
-		[DllImport(dll, CharSet = cSet, CallingConvention = call)] public static extern int    mesh_ray_intersect(IntPtr mesh, Ray model_space_ray, out Ray out_pt,IntPtr out_uv_pos);
-		[DllImport(dll, CharSet = cSet, CallingConvention = call)] public static extern int    mesh_ray_intersect(IntPtr mesh, Ray model_space_ray, out Ray out_pt, out int out_start_inds);
+		[DllImport(dll, CharSet = cSet, CallingConvention = call)] public static extern int    mesh_ray_intersect(IntPtr mesh, Ray model_space_ray, out Ray out_pt,IntPtr out_start_inds);
+		[DllImport(dll, CharSet = cSet, CallingConvention = call)] public static extern int    mesh_ray_intersect(IntPtr mesh, Ray model_space_ray, out Ray out_pt, out uint out_start_inds);
+		[DllImport(dll, CharSet = cSet, CallingConvention = call)] public static extern bool   mesh_get_triangle (IntPtr mesh, uint triangle_index, out Vertex a, out Vertex b, out Vertex c);
```

Comment:
Marshalling a `bool32_t` means an `int` is needed here. `bool` may crash certain platforms.